### ISSUE TITLE
ImportError: cannot import name 'appengine'

### ISSUE THREAD

**Issue by qvasic** [mvantellingen/python-zeep#413]
Hello!

I am trying to start using zeep, but I cannot even get past the simple example:

```
vdem@PC-vdem ~
$ python3
Python 3.6.1 (default, Mar 21 2017, 21:49:16)
[GCC 5.4.0] on cygwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import zeep
>>> c = zeep.Client( "ns.wsdl" )
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/requests_toolbelt/_compat.py", line 49, in <module>
    from requests.packages.urllib3.contrib import appengine as gaecontrib
ImportError: cannot import name 'appengine'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/site-packages/zeep/client.py", line 121, in __init__
    self.wsdl = Document(wsdl, self.transport, strict=strict)
  File "/usr/lib/python3.6/site-packages/zeep/wsdl/wsdl.py", line 78, in __init__
    root_definitions = Definition(self, document, self.location)
  File "/usr/lib/python3.6/site-packages/zeep/wsdl/wsdl.py", line 180, in __init__
    self.bindings = self.parse_binding(doc)
  File "/usr/lib/python3.6/site-packages/zeep/wsdl/wsdl.py", line 386, in parse_binding
    from zeep.wsdl import bindings
  File "/usr/lib/python3.6/site-packages/zeep/wsdl/bindings/__init__.py", line 1, in <module>
    from .soap import Soap11Binding, Soap12Binding  # noqa
  File "/usr/lib/python3.6/site-packages/zeep/wsdl/bindings/soap.py", line 4, in <module>
    from requests_toolbelt.multipart.decoder import MultipartDecoder
  File "/usr/lib/python3.6/site-packages/requests_toolbelt/__init__.py", line 12, in <module>
    from .adapters import SSLAdapter, SourceAddressAdapter
  File "/usr/lib/python3.6/site-packages/requests_toolbelt/adapters/__init__.py", line 12, in <module>
    from .ssl import SSLAdapter
  File "/usr/lib/python3.6/site-packages/requests_toolbelt/adapters/ssl.py", line 16, in <module>
    from .._compat import poolmanager
  File "/usr/lib/python3.6/site-packages/requests_toolbelt/_compat.py", line 51, in <module>
    from urllib3.contrib import appengine as gaecontrib
ImportError: cannot import name 'appengine'
>>>
```

Installing module appengine from PyPI doesn't help at all. Please advise.

Thanks!
Vasyl

**Comment by mvantellingen** [comment]
Which version of requests and requests_toolbelt do you have installed (pip freeze)

**Comment by mvantellingen** [comment]
This seems to be an issue in requests_toolbelt which might be fixed in 0.7.1. If that isn't the issue then i'm afraid I can't provide any more help on this.
